### PR TITLE
skip Lean Coffee in CW14, I'm PTO

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: other.yml [hunk 1]
```diff
@@ -52,3 +54,13 @@ events:
         - 2023-01-02 15:05:00
         - 2023-02-20 15:05:00
         - 2023-03-20 15:05:00
+  - summary: "Lean SCS Operator Coffee"
```
Comment (garloff) [inline other.yml:57]: If you start a new series on 2023-04-24, shouldn't the end date of the old series be adjusted to not overlap?

File: other.yml [hunk 2]
```diff
@@ -18,6 +18,8 @@ events:
       until: 2023-06-30  # required
```
Comment (garloff) [inline other.yml:18]: Suggest: 2023-03-31.

Comment (fkr) [reply other.yml:18]: I'm not adding a new series. I'm adding a onetime Lean Coffee (but that should probably go to the onetimes file then - I'll fix that).

**Review by itrich — APPROVED** [review]
LGTM!

**Pull request merged**